.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: node_js
 node_js:
   - "6.9"
+services:
+ - mongodb
 before_script:
   - npm install -g gulp
-script: gulp
+script:
```

Comment:
@reblace Yes, it appears so per https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step

With this approach, both steps will execute even if the first fails. Moving to a single command concatenated with "&&" appears to be the way to fail and exit on the first.